src/Microsoft.ML.StandardLearners/Standard/SdcaStatic.cs

```diff
@@ -36,7 +36,19 @@ public static partial class RegressionTrainers
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[SDCA](../../../docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs?range=5-8,12-70) "The SDCA regression example."]
+        ///  [!code-csharp[SDCA](../../../docs/samples/Microsoft.ML.Samples/Trainers.cs?range=5-8,12-70 "The SDCA regression example.")]
```

Comment:
>../../../docs/samples/Microsoft.ML.Samples/Trainers.cs [](start = 33, length = 54)

Is the same file in all three examples?

Reply:
If yes then may be make all the URLs consistent.

---
In reply to: [223510423](https://github.com/dotnet/machinelearning/pull/1184#discussion_r223510423) [](ancestors = 223510423)

Reply:
thanks for looking @zeahmed. I am tryign to see which link format is the correct one, hence the variation. This should get cleaned up right after getting one working example. 

---
In reply to: [223511069](https://github.com/dotnet/machinelearning/pull/1184#discussion_r223511069) [](ancestors = 223511069,223510423)